newdle/client/src/components/NewdleTitle.js

```diff
@@ -30,15 +31,19 @@ export default function NewdleTitle({
       <div className={styles['flexbox']}>
         <div>
           <div className={styles['title']}>
-            <h1 className={styles['header']}>{title}</h1>
+            <h1 className={styles['header']}>
+              <Trans>{title}</Trans>
```

Comment:
This one doesn't really make sense since you cannot translate a string not known in advance.

Reply:
Ah, that makes sense. I've removed it. Thanks